Code/V2/src/App.tsx

```diff
@@ -50,9 +53,23 @@ function App() {
             React Redux
           </a>
         </span>
+        <Row>
```

Comment:
Here I made use of the key component, showing all of the possible stylings it may take on. Again, this was not technically a necessary change.